src/errors.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+class ConfigError extends Error {}
```

Comment:
I seem to recall there being issues in some engines with subclassing builtins but I can't remember if it affects `Error`...

Reply:
Are you perhaps thinking about that it used to be more or less impossible to subclass builtins before ES6 (and that it therefore can’t be transpiled correctly)?

Reply:
Ah, yeah that could be it. If someone was to downlevel compile this to es5 for browser use will it still work?

I've always done it this way. Avoids nasty `instanceof` problems as well.

```js
function createConfigError(message) {
  const error = new Error(message);
  error.type = "ConfigError";
  return error;
}

function isConfigError(error) {
  return error && error.type === "ConfigError";
}
```

Reply:
Extending `Error` seems to work flawlessly in Node.js 4+. Let’s stick with that until somebody brings up an issue.